lib/vagrant-service-manager/command.rb

```diff
@@ -155,6 +158,23 @@ def find_machine_ip
         end
       end
 
+      def sha_id(file_data)
+        Digest::SHA256.hexdigest file_data
+      end
+
+      def certs_present_and_valid?(path, machine)
+        return false unless File.directory?(path)
+
+        # check validity of certs
+        Dir[path + "/*"].each do |f|
+          guest_file_path = "#{DOCKER_PATH}/#{File.basename(f)}"
+          guest_sha = machine.guest.capability(:sha_id, guest_file_path)
+          return false if sha_id(File.read(f)) != guest_sha
```

Comment:
We should be comparing sha checksum between files in #{secrets_path} and #{DOCKER_PATH} here right ?